lib/hammer/application.ex

```diff
@@ -1,68 +0,0 @@
-defmodule Hammer.Application do
-  @moduledoc """
-  Hammer application, responsible for starting the backend worker pools.
```

Comment:
some of tha doc is still valid here I think  like the optionds so we might need to move that to the hammer.ex